### ISSUE TITLE
frontend, backend option not available in 0.0.3

### ISSUE THREAD

**Issue by lucaseo** [Wirg/stqdm#5]
Hello, @Wirg 

First of all, thanks for implementing this wonderful idea.

I tried to disable backend progressbar and only allow progressbar in streamlit application.
so I passed the parameter like this
```
for idx in stqdm(range(len(data)), backend=False, frontend=True):
```

However, it showed error.
`TqdmKeyError: "Unknown argument(s): {'backend': False, 'frontend': True}"`

My current library versions are as followed.
I installed stqdm via `pip`

```
tqdm == 4.61.1
stqdm == 0.0.3
streamlit == 0.83.0
```

**Comment by Wirg** [comment]
Hello @lucaseo ,

Thanks for the issue.
I indeed never published a version of stqdm with this feature.

I just published `v0.0.4` of streamlit to `pypi` with this functionality. Tell me if it works !


Have a nice day

**Comment by Wirg** [comment]
Marking it has resolved.
Don't hesitate to reopen if you have any issues.
